### PR TITLE
[MODORDERS-845] - Release v12.6.0

### PR DESCRIPTION
**Pull request by Dmitriy-Butramyou**
## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all the appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?
- Did you modify code to call some additional endpoints?
  - [ ] If so, do you check that necessary module permission added in ModuleDescriptor-template.yaml file?

Ideally, all the PRs involved in breaking changes would be merged on the same day
to avoid breaking the folio-testing environment.
Communication is paramount if that is to be achieved,
especially as the number of inter-module and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems,
ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,9 +1,49 @@
...
+* [MODORDERS-598](https://issues.folio.org/browse/MODORDERS-598) - Replace CompletableFuture.allOf with FolioVertxCompletableFuture.allOf
+
+### Bug Fixes
+* [MODORDERS-841](https://issues.folio.org/browse/MODORDERS-841) - Encumbrance current amount is calculated incorrectly after changing fund distribution when related paid invoice exists
+* [MODORDERS-833](https://issues.folio.org/browse/MODORDERS-833) - Negative encumbrance values appear for expense classes after rollover
+* [MODORDERS-827](https://issues.folio.org/browse/MODORDERS-827) - Order view is not restricted by Acquisition unit
+* [MODORDERS-816](https://issues.folio.org/browse/MODORDERS-816) - A user with order create permission can not add suffix and prefix to the order
+* [MODORDERS-810](https://issues.folio.org/browse/MODORDERS-810) - Invoice is unable to approve due to protected fields on productIds
+* [MODORDERS-713](https://issues.folio.org/browse/MODORDERS-713) - Add prefix and suffix to PO number when create or update order
+* [MODORDERS-473](https://issues.folio.org/browse/MODORDERS-473) - Memory leak in orders app
```
Comment (grabsefx) [inline NEWS.md:45]: no changes in master branch regarding MODORDERS-473. there were only perfomance checks

Comment (Dmitriy-Butramyou) [reply NEWS.md]: I deleted this task from news

**Review by SerhiiNosko — APPROVED** [review]

File: NEWS.md [hunk 2]
```diff
@@ -1,9 +1,48 @@
...
+* [MODORDERS-820](https://issues.folio.org/browse/MODORDERS-820) - Introduce "deleteHolding" param to orders put endpoint
+* [MODORDERS-825](https://issues.folio.org/browse/MODORDERS-825) - Align the module with API breaking change
+* [MODORDERS-823](https://issues.folio.org/browse/MODORDERS-823) - Implement logic that defines which event should be issued as a result of saving Pending Orders
+* [MODORDERS-821](https://issues.folio.org/browse/MODORDERS-821) - Implement "Create new holdings for location" flag
+* [MODORDERS-817](https://issues.folio.org/browse/MODORDERS-817) - Associate mapped POLines with single Order based on POLine limit value
+* [MODORDERS-808](https://issues.folio.org/browse/MODORDERS-808) - Create "OrdersPostProcessingEventHandler" for mod-orders
+* [MODORDERS-807](https://issues.folio.org/browse/MODORDERS-807) - Copy number does not get applied to item when creating item through receiving app
+* [MODORDERS-805](https://issues.folio.org/browse/MODORDERS-805) - Replace FolioVertxCompletableFuture usage - merge and resolve merge conflicts
```
Comment (SerhiiNosko) [inline NEWS.md:21]: I think we can delete this resolve and merge conflicts story from the list

File: NEWS.md [hunk 3]
```diff
@@ -1,9 +1,48 @@
...
+* [MODORDERS-784](https://issues.folio.org/browse/MODORDERS-784) - Logging improvement - Configuration
+* [MODORDERS-783](https://issues.folio.org/browse/MODORDERS-783) - Implement Orders Export History API
+* [MODORDERS-780](https://issues.folio.org/browse/MODORDERS-780) - Replace FolioVertxCompletableFuture usage
+* [MODORDERS-775](https://issues.folio.org/browse/MODORDERS-775) - Add deduplication mechanism for Create Order event
+* [MODORDERS-773](https://issues.folio.org/browse/MODORDERS-773) - Implement action profile handler for order creation
+* [MODORDERS-724](https://issues.folio.org/browse/MODORDERS-724) - Replace use of deprecated class HttpClientFactory
+* [MODORDERS-696](https://issues.folio.org/browse/MODORDERS-696) - Replace all utility methods for HTTP calls from HelperUtils
+* [MODORDERS-647](https://issues.folio.org/browse/MODORDERS-647) - Implement Find or create logic for Holdings generated through the GOBI API
+* [MODORDERS-598](https://issues.folio.org/browse/MODORDERS-598) - Replace CompletableFuture.allOf with FolioVertxCompletableFuture.allOf
```
Comment (grabsefx) [inline NEWS.md:36]: should be deleted as well. all work has been done in MODORDERS-780

Comment (grabsefx) [inline NEWS.md:36]: should be deleted as well. all work has been done in MODORDERS-780

**Review by grabsefx — APPROVED** [review]

**Pull request merged**